tests/integration/test_cross_cli_live_tmux.py

```
@@ -0,0 +1,90 @@
+
+    def tearDown(self) -> None:
+        shutil.rmtree(self.tmp, ignore_errors=True)
+
+    def _spawn(self, cli: str, prompt: str, timeout: int = 90) -> dict:
+        # Spawn a real session; the Stop hook fires when the session ends
+        # (or when the tmux pane is closed). The hook writes last.json
+        # before the session exits cleanly.
+        bin_ = shutil.which(cli)
+        if not bin_:
+            self.skipTest(f"{cli} not on PATH")
+        proc = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)